contributing/workflow.md

```diff
@@ -215,6 +214,33 @@ Label your PRs with the one of the following [labels](https://github.com/codeign
 And if your PRs have the breaking changes, label the following label:
 - **breaking change** ... PRs that may break existing functionalities
 
+## Updating Your Branch
+
+If you are asked for changes in the review, commit the fix in your branch and push it to GitHub again.
+
+If the `develop` branch progresses and conflicts arise that prevent merging, or if you are asked to *rebase*,
+do the following:
+
+Synchronize your repository:
+
+    git checkout develop
+    git fetch upstream
+    git merge upstream/develop
+    git push origin develop
```

Comment:
I prefer to keep it simple, unless you think the verbosity adds something.
```suggestion
    git pull upstream develop
    git push
```

Reply:
I strongly oppose it. I think the verbosity adds values.
This document is for git beginners. 
Omitted commands hide what they are doing.

`git pull` is harmful.
https://stackoverflow.com/questions/15316601/in-what-cases-could-git-pull-be-harmful

`git push` does not show where and what you push.

Reply:
@MGatner I've been using git lately. I used to the desktop version of GitHub. In the desktop version, I used pull. `Pull` created  problems for me. I am generally opposed to this change.